flow/size.yaml

```diff
@@ -62,3 +83,107 @@ components:
           type: integer
           default: 1518
           x-field-uid: 2
+    Flow.SizeWeightPairs:
+      type: object
+      description: |-
+        Custom frame size distribution, defined as <size, weight> pairs (including IMIX distribution).
+      properties:
+        choice:
+          type: string
+          default: predefined
+          x-field-uid: 1
+          x-enum:
+            predefined:
+              x-field-uid: 1
+            custom:
+              x-field-uid: 2
+        predefined:
+          type: string
+          default: imix
+          description: |-
+            Option allow to specify a predefined frame size distribution <size, weight> pairs (including IMIX distribution).  
```

Comment:
I think we can start with "Specify pre-defined frame size ...

Reply:
done